trackpy/tests/test_misc.py

```diff
@@ -60,3 +63,27 @@ def test_convenience_funcs(self):
         self.assertEqual(len(trackpy.logger.handlers), 1)
         self.assertEqual(trackpy.logger.level, logging.INFO)
         self.assertEqual(trackpy.logger.propagate, 1)
+
+
+class NumbaTests(StrictTestCase):
+    def setUp(self):
+        if not NUMBA_AVAILABLE:
+            raise nose.SkipTest("Numba not installed. Skipping.")
+        self.funcs = trackpy.try_numba._registered_functions
+
+    def test_registered_numba_functions(self):
+        self.assertGreater(len(self.funcs), 0)
+
+    def test_enabled(self):
+        trackpy.enable_numba()
+        for registered_func in self.funcs:
+            module = __import__(registered_func.module_name, fromlist='.')
+            func = getattr(module, registered_func.func_name)
+            self.assertIs(func, registered_func.compiled)
+
+    def test_disabled(self):
```

Comment:
Great! Do we need to re-enable numba at the end of this test? Otherwise I think it could affect subsequent tests.